troncos/tracing/__init__.py

```diff
@@ -13,15 +13,15 @@ def configure_tracer(
     enabled: bool,
     endpoint: str,
     service_name: str,
-    service_attributes: dict[str, Any] | None = None,
+    resource_attributes: dict[str, Any] | None = None,
```

Comment:
This will break the api, so we need to do a major release for this change. I like that the api will be consistent with the visualization in grafana.

Reply:
I will probably also use the opportunity to make other minor breaking api changes before we release.

Reply:
Yes

Can we add `span attribute` as a argument somewhere as well. 

Hiding the ddtrace set tags config?

Reply:
Experience has showed that wrapping the ddtrace api is not popular. So I would just let it be :)